homeassistant/components/bbox/sensor.py

```diff
@@ -126,6 +128,12 @@ def update(self):
             self._state = round(self.bbox_data.data["rx"]["bandwidth"] / 1000, 2)
         elif self.type == "current_up_bandwidth":
             self._state = round(self.bbox_data.data["tx"]["bandwidth"] / 1000, 2)
+        elif self.type == "uptime":
+            self._state = str(
+                timedelta(seconds=self.bbox_data.router_infos["device"]["uptime"])
```

Comment:
timedelta are not allowed in the state machine. You are only allowed UTC timestamps and then set the device class to timestamp.